index.html

```diff
@@ -35,6 +35,20 @@
             des <a href="http://codefor.de/berlin">Open Knowledge Lab Berlin</a>.
             <span id="dataSource">Datenquelle</span>. <a href="https://github.com/wo-ist-markt/wo-ist-markt.github.io">Code</a> auf GitHub.</p>
         </div>
+        <!-- Piwik -->
+        <script type="text/javascript">
+          var _paq = _paq || [];
+          _paq.push(['trackPageView']);
+          _paq.push(['enableLinkTracking']);
+          (function() {
+            var u="//piwik.okfn.de/";
+            _paq.push(['setTrackerUrl', u+'piwik.php']);
+            _paq.push(['setSiteId', 18]);
```

Comment:
It is the 18th site that is tracked on that piwik instance as far as I can tell. So this is out of our control.